src/content/docs/guides/policy-diff.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Reviewing Policy
+description: How to review LavaMoat Policy and its diffs
+---
+
+<!-- markdownlint-disable no-inline-html -->
+
+This guide will help you review your policy on a continuous basis.
+
+## Why review Policy?
+
+The Policy File generated by LavaMoat is based on a scan of your codebase, identifying all the powers it uses. The initial policy doesn't provide security on its own. Instead, it's your review of the initial policy and the subsequent updates (with new or updated dependencies) that makes your application secure.
+
+Reviewing diffs as you go lets you spot suspicious packages or limit the powers you wish newly added packages can use.
+
+The review of your initial policy may seem like a lot of effort, but it's an investment in a good starting point for your application.  
+If you generate the initial policy from a known uncompromised version of your application the main purpose of the initial review is to potentially take away some of the excessive powers from packages you don't use in their full capacity.
+
+## How to review your policy?
+
+LavaMoat Policy is listing all powers that a package can use. That's what you'll find in `globals` and `builtin` fields.
+It also lists which other packages are allowed for the current package to import. You can follow those relations to see whether a package with access to very [powerful APIs](#powerful-apis) is used by any suspicious packages as a dependency. See [Principle of Least Authority][PoLA]
+
+### What to look for when reviewing a Policy diff?
+
+The goal of rviewing the diff is to spot a malicious package being added.
+
+#### TL;DR
+- check `globals` and `builtins` for new powers and investigate if you're surprised the package would need them
+- check if new relationships in `packages` are pointing to packages with very [powerful APIs](#powerful-apis) (eg. spawning child processes in Node.js)
+- be aware that the identifier may change to `pkgC>actual-name` from `pkgB>pkgA>actual-name` BUT! If the package now also has totally different powers it's likely a different package of the same name. Investigate! `npm ls actual-name` should help.
```

Comment:
```suggestion
- be aware that the identifier may change to `pkgC>actual-name` from `pkgB>pkgA>actual-name` BUT! If the package now also has totally different powers, it's likely a different package of the same name. Investigate! `npm ls actual-name` should help.
```

Reply:
worth mentioning `yarn why actual-name` for repos using Yarn?

Reply:
I did that for the allow-scripts readme and regretted it later. 
I could mention `yarn list` tho